src/Gir/Marshal/SymbolTable.Builtin.cs

```diff
@@ -41,7 +40,42 @@ void RegisterPrimitives ()
 			AddType(new Primitive("gdouble", "double", "0.0"));
 			AddType(new Primitive("double", "double", "0.0"));
 			AddType(new Primitive("goffset", "long", "0"));
+
+			AddType(new Primitive("ssize_t", "IntPtr", "IntPtr.Zero"));
+			AddType(new Primitive("gssize", "IntPtr", "IntPtr.Zero"));
+			AddType(new Primitive("size_t", "UIntPtr", "UIntPtr.Zero"));
+			AddType(new Primitive("gsize", "UIntPtr", "UIntPtr.Zero"));
+
+			// FIXME: Add these eventually.
+			//AddType(new MarshalGen("time_t", "System.DateTime", "IntPtr", "GLib.Marshaller.DateTimeTotime_t ({0})", "GLib.Marshaller.time_tToDateTime ({0})"));
+
+			//AddType(new ConstStringGen("const-gchar"));
```

Comment:
Not sure about "const-" prefix here

Reply:
It's commented out. Needs to be fixed when imported with proper marshalling.

Reply:
Yeah, I was just making sure we do not forget about that ;)